lib/nixd/src/Diagnostic.cpp

```diff
@@ -16,7 +16,7 @@
   FUNC(ANSI_MAGENTA)                                                           \
   FUNC(ANSI_CYAN)
 
-static std::string stripANSI(std::string Msg){
+static std::string stripANSI(std::string Msg) {
```

Comment:
you can ignore this stuff, because source files are formatted in other PRs then. Actually this is changed in #50 and you will introduce git conflicts caused by this diff.